reference/forms/types/birthday.rst

```diff
@@ -1,13 +1,13 @@
 .. index::
    single: Forms; Fields; birthday
 
-birthday Field Type
+Birthday Field Type
```

Comment:
Yep, we use `birthday` because it's basically `birthday` is the case-sensitive field type name. So I think we need to keep this lowercased.